src/WebJobs.Extensions.DurableTask/AzureStorageDurabilityProvider.cs

```diff
@@ -208,12 +222,30 @@ internal static OrchestrationInstanceStatusQueryCondition ConvertWebjobsDurableC
             string connectionName,
             out IScaleMonitor scaleMonitor)
         {
+            var storageAccount = this.storageAccountProvider.GetStorageAccountDetails(connectionName).ToCloudStorageAccount();
+            var metricsProvider = this.GetMetricsProvider(functionName, hubName, storageAccount, this.logger);
```

Comment:
Please use explicit variable types instead of `var` since we can't infer the type by simply reading the code.

Reply:
Done!